ci-operator/config/red-hat-data-services/openvino_model_server/red-hat-data-services-openvino_model_server-v2022.3-release.yaml

```diff
@@ -229,6 +235,37 @@ resources:
     requests:
       cpu: "6"
       memory: 12Gi
+tests:
+- as: unit
```

Comment:
OVMS doesn't need the make test part, can remove it, the mirroring is great though!

Reply:
done